concordium-sdk-examples/README.md

```diff
@@ -0,0 +1,13 @@
+# Examples for [Concordium SDK](../concordium-sdk/)
+This project comprises for CLI applications in the [examples](./src/main/java/com/concordium/sdk/examples/) directory. Each file is an independent CLI application which acts as an example for various methods [Concordium GRPC client](../concordium-sdk/src/main/java/com/concordium/sdk/ClientV2.java) implements
+
+## Running Cli Applications
+Each cli application can be run by standard procedures to run any java application with allows for specifying parameters. Example
+
+```bash
+cargo build --manifest-path ./crypto-jni/Cargo.toml --release
```

Comment:
can't one just `make` instead of building and copying manually?

Reply:
yes surely. its done, thanks